.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/MadKudu/madkudu.js
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
```

Comment:
Sorry for the back and forth, but here I might leave a comment saying that this environment variable is inherited from the migration tool (if it indeed is). That way you know why it's there and can delete it later if necessary

Reply:
Note: this may actually be needed, in which case ignore particular case, but general advice still applies :)